sklearn/datasets/base.py

```diff
@@ -255,14 +255,25 @@ def load_iris():
 
     Read more in the :ref:`User Guide <datasets>`.
 
+    Parameters
+     ----------
+    return_X_y : boolean, default=False.
+        If True, returns (data, target) instead of a Bunch object.
+        See below for more information about the `data` and `target` object.
+
     Returns
     -------
+    if return_X_y == false
     data : Bunch
         Dictionary-like object, the interesting attributes are:
         'data', the data to learn, 'target', the classification labels,
         'target_names', the meaning of the labels, 'feature_names', the
         meaning of the features, and 'DESCR', the
         full description of the dataset.
+    
+    if return_X_y == true
+    (data,target) : tuple
```

Comment:
Hey, I'm a little confused here. The methods and description doesn't seem to be updated after I ran the command.

Path of html page: ```file:///home/manu/github/scikit-learn/doc/_build/html/stable/modules/generated/sklearn.datasets.load_iris.html#sklearn.datasets.load_iris```

![image](https://cloud.githubusercontent.com/assets/12952080/16981428/32af687e-4e88-11e6-8b33-ab92e6bfb977.png)